tests/sanity/ignore-2.13.txt

```diff
@@ -1,2 +1,3 @@
 .azure-pipelines/scripts/publish-codecov.py replace-urlopen
+plugins/modules/docker_compose_v2.py validate-modules:return-syntax-error
```

Comment:
what in this module triggers a return syntax error?

Reply:
It's the `type: raw` IIRC. Some older versions might also barf on `choices` in return values.